ansible/roles/ansible-horizon-plugin/tasks/pre_tasks.yml

```diff
@@ -2,6 +2,27 @@
 #For Debian: apache2
 #For RHEL: httpd
 
+- name: Find Horizon directory path
+  find: 
+    paths: "{{ horizon_virtual_env | dirname }}/"
+    file_type: directory
+    use_regex: yes
+    patterns:
+      - '^.*?horizon*'
+    recurse: no
+  register: horizon_path
+
+- name: Find Horizon directory path & count
+  set_fact:
+    horizon_dir_path: "{{ horizon_path.files| map(attribute='path')| list|join(',') }}"
+    horizon_dir_cnt: "{{ horizon_path.files| map(attribute='path')| list|length }}"
```

Comment:
it returns one attribute `matches` which gives you the number of matches it found. You can use that number as it is, no need to process the returned file paths